npm/design-system/src/css/derived/types.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+import type colors from './jsColors.scss'
+import type { ClassNames as JsSpacing } from './jsSpacing.scss'
+import type { ClassNames as JsTypography } from './jsTypography.scss'
+import type {ClassNames as JsSurface } from './jsSurfaces.scss'
+
+type Digit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
+type TwoDigit = `${Digit}${Digit}`
```

Comment:
Ah, this is the new TS feature you wanted?

Reply:
This is indeed the TS feature, among a few other things.

Reply:
I'm constantly amazed at how far TS has come (and how it keeps evolving).